distributed/worker.py

```diff
@@ -1837,7 +1839,7 @@ def transition_ready_error(self, ts):
             assert ts.traceback is not None
         self.send_task_state_to_scheduler(ts)
 
-    def transition_ready_memory(self, ts, value=None):
+    def transition_ready_memory(self, ts, value=no_value):
         if value:
```

Comment:
Sorry, I should have included this with my earlier comment

```suggestion
        if value is not no_value:
```

Reply:
Thanks, I should've caught that.